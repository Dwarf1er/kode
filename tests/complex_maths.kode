SHOW 1 PLUS 2 PLUS 3.
SHOW 2 MINUS 2 MINUS 3.
SHOW 3 PLUS 2 MINUS 3 PLUS 4.
SHOW 2 TIMES 2 PLUS 3 TIMES 3.
SHOW 4 DIVIDE 2 PLUS 3 TIMES 2 MINUS 4.
SHOW 4 PLUS 2 TIMES 3 MOD 3 MINUS 4 DIVIDE 2.
SHOW 4 PLUS 2 MINUS 3 PLUS 2 MINUS 4 PLUS 4.
