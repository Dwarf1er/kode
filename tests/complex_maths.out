6
-3
6
13
4
2
5
